woocommerce/woocommerce-android
# [Support Requests] Create Form UI

Summary
==========
Fix issue #8331 by introducing our own Support Request form view that will replace the current usage of the Zendesk Activities. Since the Settings and Help section of the app still doesn't have proper support for navigation graphs, the form was implemented through a new Activity instead of a Fragment.

The original designs were made with only the iOS version in mind, so I took the liberty of adapting the form to fit our view style around WCAndroid app. Please feel free to suggest any adjustments.


<details>
  <summary>iOS UI implementation</summary>

| Top  | Bottom |
| ------------- | ------------- |
| ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-21 at 20 09 26](https://user-images.githubusercontent.com/5920403/220479638-e65050ac-3e36-40af-9313-00f5faa4c82a.png) | ![Simulator Screen Shot - iPhone 14 Pro - 2023-02-21 at 20 09 28](https://user-images.githubusercontent.com/5920403/220479641-01a75663-1c39-4971-ba1e-dba8d8b0f225.png) |

</details>

⚠️⚠️⚠️ Disclaimer
==========
This PR only introduces the form UI, no ticket creation event should be triggered, and the submit support request button is disabled to avoid misclicks while the feature is under development. The button is connected with the Zendesk creation event, but please bear in mind that overriding the button state and creating a ticket may create unwanted results in our Zendesk woo mobile queue. 

Captures
==========
https://user-images.githubusercontent.com/5920403/220478309-e51fd4e7-1d15-4b45-b059-10b8b48dc234.mp4

How to Test
==========
1. Open the app and hit the Menu section in the bottom bar
2. Once inside the menu, hit the `gear` icon to go to the app settings
3. Select the `Help & support` option of the app settings list, it's the very first option
4. Inside the Help options, select the `Contact support` option
5. Verify that the new form opens and all expected UI interactions work as expected through all view elements, and only the `submit` button is disabled. Validation constraints will be added later.

Update release notes:
